homeassistant/components/abode/alarm_control_panel.py

```diff
@@ -23,12 +23,11 @@
 
 
 async def async_setup_entry(hass, config_entry, async_add_entities):
-    """Set up an alarm control panel for an Abode device."""
+    """Set up Abode alarm control panel device."""
     data = hass.data[DOMAIN]
-
-    async_add_entities(
-        [AbodeAlarm(data, await hass.async_add_executor_job(data.abode.get_alarm))]
-    )
+    entity = [AbodeAlarm(data, await hass.async_add_executor_job(data.abode.get_alarm))]
+    data.entities.extend(entity)
```

Comment:
Remove this.

Add the entity_id to a dictionary instead from `async_added_to_hass` entity coroutine method.

Reply:
Is there an expected or standardized way to store a references to `entity_id` within `self.hass.data[DOMAIN]`? Looking at the tuya integration, I see this in the `async_added_to_hass` coroutine:
````
self.hass.data[DOMAIN]["entities"][dev_id] = self.entity_id
````
I'll have to think about this with the Abode component if it has to be stored in a dictionary since we don't have anything like a `dev_id` within Abode.

Reply:
You can store a `set` if there is no key value pair.